ci/lint-docs.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+### This script should only be run on Improbable's internal build machines.
+### If you don't work at Improbable, this may be interesting as a guide to what software versions we use for our
+### automation, but not much more than that.
+
+set -e -u -x -o pipefail
+
+if [[ -z "$BUILDKITE" ]]; then
+  echo "This script is only to be run on Improbable CI."
+  exit 1
+fi
+
+cd "$(dirname "$0")/../"
+
+ci/bootstrap.sh
```

Comment:
Do we need this here? I think we run `bootstrap.sh` on every build during the `post-checkout` right?

Reply:
we haven't set up a post-checkout for the docs branch - didn't think it was necessary as this is the only step that requires bootstrap to have run

Reply:
THat's probably fine then